README.md

```diff
@@ -1,76 +1,55 @@
+<a href="https://hub.docker.com/r/jellyfin/jellyfin"><img alt="Docker Pull Count" src="https://img.shields.io/docker/pulls/jellyfin/jellyfin.svg"></a>
+<a href="https://matrix.to/#/#jellyfin:matrix.org"><img alt="Chat on Matrix" src="https://img.shields.io/matrix/!JXmoIxArWgVvbKKqex/matrix.org.svg?logo=matrix"></a>
+<a href="https://www.reddit.com/r/jellyfin/"><img alt="Join our Subreddit" src="https://img.shields.io/badge/reddit-r%2Fjellyfin-%23FF5700.svg"></a>
+</p>
 
-### Arch
+For further details, please see [our wiki](https://github.com/jellyfin/jellyfin/wiki).
 
-The Jellyfin package is in the AUR at https://aur.archlinux.org/packages/jellyfin-git/
+To receive the latest project updates feel free to join [our public chat on Matrix/Riot](https://matrix.to/#/#jellyfin:matrix.org), and follow us on [Social Media](https://github.com/jellyfin/jellyfin/wiki/Social-Media).
 
-### Unraid
+## About
 
-An Unraid Docker template is available. See [this documentation page](https://github.com/jellyfin/jellyfin/blob/master/unRaid/docker-templates/README.md) for details on installing it.
+The Jellyfin project was started as a result of Emby's decision to take their code closed-source, as well as various philosophical differences with the core developers. Jellyfin seeks to be the free software alternative to Emby and Plex to provide media management and streaming from a dedicated server to end-user devices.
 
-### Debian/Ubuntu
-
-A package repository is available at https://repo.jellyfin.org.
-
-NOTE: Ubuntu users may find that the `ffmpeg` dependency package is not present in their release or is simply a rebranded `libav` which is not directly compatible. Please [obtain the ffmpeg package directly from the FFMPEG site](https://ffmpeg.org/download.html#build-linux) to use Jellyfin on Ubuntu.
-
-For instructions on using the repository, please see [our wiki](https://github.com/jellyfin/jellyfin/wiki/Jellyfin-Debian-repository).
-
-## Building Jellyfin packages from source
-
-Jellyfin seeks to integrate build facilities for any desired packaging format. Instructions for the various formats can be found below.
-
-NOTE: When building from source, only cloning the full Git repository is supported, rather than using a `.zip`/`.tar` archive, in order to support submodules.
-
-### Debian/Ubuntu
+Jellyfin is descended from Emby 3.5.2, ported to the .NET Core framework, and aims to contain build facilities for every platform.
 
-Debian build facilities are integrated into the repo at `debian/`.
+## Issues and Feature Requests
 
-0. Install the `dotnet-sdk-2.2` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.2).
-0. Run `dpkg-buildpackage -us -uc`.
-0. Install the resulting `jellyfin_*.deb` file on your system.
 
-A huge thanks to Carlos Hernandez who created the original Debian build configuration for Emby 3.1.1.
 
-### Windows (64 bit)
+<p align="center">
```

Comment:
And these I suggest to align to the left as they were (if headlines are centered), but that's up to you.

Reply:
Hopefully the new style makes more sense.